src/shared/config/firebase.ts

```
@@ -12,21 +26,93 @@ const firebaseConfig = {
+/**
+ * Firebase Analytics 초기화 함수
+ *
+ * 초기화 과정:
+ * 1. 브라우저 환경 확인 (window 객체 존재)
+ * 2. Analytics 지원 여부 확인 (isSupported)
+ * 3. Analytics 인스턴스 생성 및 반환
+ *
+ * @returns {Promise<Analytics | null>} 초기화된 Analytics 인스턴스 또는 null
+ */
+const initAnalytics = async () => {
+  if (typeof window !== 'undefined') {
```

ESLint

[error] Replace `'undefined'` with `"undefined"`

(prettier/prettier)